src/timeMachine/components/RawFluxDataGrid.tsx

```diff
@@ -119,6 +119,9 @@ export default class extends PureComponent<Props, State> {
 
   private getCellData(row, column) {
     const {data} = this.props
-    return data[row][column]
+    if (data[row] && data[row][column]) {
+      return data[row][column]
+    }
+    return ''
```

Comment:
This was added to address a bug where the UI pagination for Notebooks has more rows than data